.ci-builds/.azure-pipelines.yml

```diff
@@ -451,8 +451,14 @@ jobs:
         %CYG_ROOT%\\bin\\sh -lc 'echo Hello' && %CYG_ROOT%\\bin\\sh -lc 'uname -a'
       displayName: 'Check Cygwin environment'
     - script: |
-        %CYG_ROOT%\\bin\\bash -lc "cd $(cygpath \"$(Build.SourcesDirectory)\") && cd icu4c/source && ./runConfigureICU Cygwin && make check"
-      displayName: 'Build and test'
+        %CYG_ROOT%\\bin\\bash -lc "cd $(cygpath \"$(Build.SourcesDirectory)\") && cd icu4c/source && ./runConfigureICU Cygwin && make"
+      displayName: 'Build ICU'
+      env:
+        CC: gcc
+        CXX: g++
+    - script: |
+        %CYG_ROOT%\\bin\\bash -lc "cd $(cygpath \"$(Build.SourcesDirectory)\") && cd icu4c/source && make check"
+      displayName: 'Run Tests'
```

Comment:
nit: Technically this should probably be something like "Build and Run Tests", as I think `make` only builds the ICU library code, and doesn't actually build any of the test code.
So if we wanted to we could use `make test` to build both the library code and the tests, and then run `make check` to execute the tests. :)

Reply:
Good point, I'll split the test build as well. :)

Reply:
Sounds good, thanks!